include/assembler.cpp

```diff
@@ -66,7 +78,9 @@ void assembleLine(std::string line, int lineNum, std::string romName) {
             break;
         }
         else if(instructions[i].asmInstruction != instruction && i == (sizeof(instructions)/sizeof(Chip8Instruction))-1) {
-            printf("Invalid instruction on line %d!\n", lineNum);
+            std::cout << "Invalid instruction on line " << lineNum << "!" << std::endl;
```

Comment:
??? Ar you mad kos? 😂 So you are passing through `lineNum` just to print an error and then immediately `return 1;` to indicate an error - which then needs to be handled anyway by the code that calls this function... 😅😏 Fix it.